### ISSUE TITLE
Snappy-java-1.1.10.1.jar library (libsnappyjava.so) fails on IBM PowerPC RedHat 8 OS with the error  GLIBC_2.32 not found (required by /libsnappyjava.so)

### ISSUE THREAD

**Issue by vineshcpaul** [xerial/snappy-java#512]
We use snappy-java-1.1.10.1.jar in one of our application that runs on IBM PowerPC architecture ( RedHat 8.6 OS) and very recently the application started failing with the following error

`/lib64/libc.so.6: version GLIBC_2.32 not found (required by /libsnappyjava.so)`

Is it because the PPC64LE version of the shared object is compiled  on a machine that has an higher version of GLIBC ( Probably RedHat 9 ) when compared to the same library on x86?

```
[jenkins@redhat8-builds ppc64le]$ strings org/xerial/snappy/native/Linux/ppc64le/libsnappyjava.so | grep -i glibc
GLIBC_2.17
GLIBC_2.32
GLIBC_2.34
[jenkins@redhat8-builds ppc64le]$ cd ../x86_64/

[jenkins@redhat8-builds- x86_64]$ strings org/xerial/snappy/native/Linux/x86_64/libsnappyjava.so | grep -i glibc
GLIBC_2.3
GLIBC_2.2.5
GLIBC_2.3.2
[jenkins@redhat8-builds- x86_64]$ 
```

**Comment by vineshcpaul** [comment]
Hi @xerial ,

Thanks for the response and suggestion. Submitted a fix similar to issue #481. 
Could you please review the PR when you get some time ?

**Comment by xerial** [comment]
dockcross now provides a LTS version of cross-compiler, which works for GLIBC 2.28, so this problem will be fixed in the next release #516
